spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js

```diff
@@ -54,12 +57,46 @@ export default class DialPad extends React.Component {
             formattedPhone: ''
         };
 
-        this._asYouType = new AsYouType('US');
+        this._createAsYouType(props.countryCode);
 
         this._onChange = this._onChange.bind(this);
         this._onSubmit = this._onSubmit.bind(this);
     }
 
+    /**
+     * Initializes the {@code AsYouType} utility for phone number fomratting with the given country code.
+     *
+     * @param {string} countryCode - ISO 3166-1 alpha-2 country code.
+     * @private
+     * @returns {void}
+     */
+    _createAsYouType(countryCode) {
+        logger.log(`Init dial pad with country code: ${countryCode}`, { countryCode });
```

Comment:
nit: should not include countryCode in the log string as it's already reported in the context object.

Reply:
Actually I did this intentional, because Im lazy to unwrap logs when reading them. I did this in few other places recently, so if you are really against that I will stop doing it.

Reply:
We were supposed to use a constant string for the log message for some querying purpose; that's what I was told anyway. Let's just go with what's easier for now.